src/main/java/com/artipie/http/DockerRoutingSlice.java

```diff
@@ -63,10 +71,20 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
         if (matcher.matches()) {
             final String group = matcher.group(1);
             if (group.isEmpty() || group.equals("/")) {
-                rsp = new BasicAuthSlice(
+                rsp = new BasicAuthzSlice(
                     new BaseEntity(),
                     this.settings.authz().authentication(),
-                    user -> !user.equals(Authentication.ANY_USER)
+                    new OperationControl(
+                        user -> {
+                            // @checkstyle NestedIfDepthCheck (10 lines)
+                            PermissionCollection res = new FreePermissions();
```

Comment:
Can't understand what does this mean?

Reply:
the same as was `user -> !user.equals(Authentication.ANY_USER)`:
`v2` endpoint (docker root) is available for any authenticated user

Reply:
> the same as was `user -> !user.equals(Authentication.ANY_USER)`: `v2` endpoint (docker root) is available for any authenticated user

This means if the user is authenticated, he has the permission to pull any image. Right?